docs/source/guide/testplan.rst

```diff
@@ -31,7 +31,7 @@ Other Test Plans can be searched via the **SEARCH** item in the **Main menu**:
    |Click column to sort by.|
 
 
-.. note::
+.. _note:
```

Comment:
`.. note:` is a Sphinx directive. `.. _cloning-testplan:` is an anchor so we can link to this section of the document from elsewhere.

Revert this change.

Reply:
Then we need to disable this check in doc8 as it complained about it.

Reply:
Are you sure you are using the latest & greatest of versions ? I don't see any complains from doc8 on the master branch which is without your change https://travis-ci.org/kiwitcms/Kiwi/jobs/637678567. 

Make sure to revert this lline.

Reply:
Done.